python/lsst/ip/isr/isrTask.py

```diff
@@ -2127,6 +2133,42 @@ def maskEdges(self, exposure, numEdgePixels=0, maskPlane="SUSPECT"):
                 maskBitMask
             )
 
+    def maskGrowTophat(self, exposure, boxRadius=0, boxThreshold=0.25, maskPlane="BAD", maskLabel="BAD"):
+        """Mask regions around large masked regions.
+
+        Parameters
+        ----------
+        exposure: `lsst.afw.image.Exposure`
+            Exposure to process.
+        boxRadius : `int`, optional
+            Half-size of the box to use to grow masks.
+        boxThreshold : `float`, optional
```

Comment:
Do you think `boxThreshold` seems like it should be a new IsrTask config parameter?

Reply:
The new mask growth radius is used, IsrConfig.brighterFatterGrowMask.

Reply:
IsrConfig.brighterFatterMaskGrowSize is the new config option, to be more explicit what it does.